### PR TITLE
Pr/1

### PR DESCRIPTION
**Pull request by Wenish**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build": "google-closure-compiler --js=src/index.js --js_output_file=dist/simpleTypeCheck.min.js --language_out=ECMASCRIPT3 --assume_function_wrapper",
     "travis-deploy-once": "travis-deploy-once",
     "semantic-release": "semantic-release",
-    "commit": "sgc"
+    "commit": "sgc",
+    "mocha": "mocha \"test/**/*.spec.js\""
```
Comment (RaphaelHaettich) [inline package.json:11]: test script dopplet, het test und mocha

**Review by RaphaelHaettich — CHANGES_REQUESTED** [review]
und commits in the future pls with ```npm run commit```, wege sematic-versioning u so

File: package.json (hunk 1 above)
Comment (Wenish) [reply package.json:11]: because tests are not only unit tests

**Review by RaphaelHaettich — APPROVED** [review]

**Comment by Wenish** [comment]
report for not merge

**Comment by RaphaelHaettich** [comment]
chentsch ne ja au selber merge ;)

**Pull request merged**